dashboards/all_metrics/modules/v3/chain_perp_stats.py

```diff
@@ -142,6 +156,22 @@ def make_charts(data):
             if st.session_state.chain.startswith("base")
             else None
         ),
+        "account_activity_daily": chart_bars(
+            data["account_activity"],
+            x_col="ts",
+            y_cols=["new_accounts_daily", "returning_accounts_daily"],
+            title="Daily New/Returning Accounts",
+            y_format="#",
+            help_text="Number of daily new/returning accounts that have at least one order settled",
+        ),
+        "account_activity_monthly": chart_area(
+            data["account_activity"],
+            x_col="ts",
+            y_cols=["new_accounts_monthly", "returning_accounts_monthly"],
+            title="Monthly New/Returning Accounts",
+            y_format="#",
+            help_text="Number of new/returning accounts that have at least one order settled in the last 28 days",
+        ),
```

Comment:
Looks perfect but one small nitpick, can we add totals to the tooltips here?

Reply:
Added!